lib/configuration.yaml

```diff
@@ -14907,6 +14907,30 @@ devnet: &devnet
     networkDiameter: 18 # should be slightly less than slot duration
     recoveryHeadersMessage: 200 # should be greater than k
 
+devnet_staging_win64:
+  <<: *devnet
+  update:
+    <<:
+    applicationName: csl-daedalus
+    applicationVersion: 0
+    lastKnownBlockVersion:
+      bvMajor: 0
+      bvMinor: 0
+      bvAlt: 0
+    systemTag: win64
+
+devnet_staging_macos64:
```

Comment:
Why **staging**? Why not `devnet_macos64`? Original configuration is called `devnet`. And it doesn't make sense to have a staging for `devnet` AFAIU.